### ISSUE TITLE
Composing command does not show double quotes

### ISSUE THREAD

**Issue by paulz** [sersoft-gmbh/xcodebuild-action#70]
### `Composing command` output is very useful for debugging failing tests locally.
Usually we copy it from log to paste to terminal locally. 

Unfortunately it fails due to double quotes:

https://github.com/Lucra-Sports/ExploratoryTests/runs/6053712764?check_suite_focus=true#step:5:13

shows:
`xcodebuild -project Experiments.xcodeproj -scheme Experiments -destination platform=iOS Simulator,name=iPhone 12 -resultBundlePath /Users/runner/work/ExploratoryTests/ExploratoryTests/test-results/all-tests test | xcpretty --color`

while actual command should be:
`xcodebuild -project Experiments.xcodeproj -scheme Experiments -destination "platform=iOS Simulator,name=iPhone 12" -resultBundlePath /Users/runner/work/ExploratoryTests/ExploratoryTests/test-results/all-tests test | xcpretty --color`

where double quotes should be around long arguments: `"platform=iOS Simulator,name=iPhone 12"`

### Second question:
Would also be possible to path for the `resultBundlePath` as it is in the workflow file?
For example:
instead of: `/Users/runner/work/ExploratoryTests/ExploratoryTests/test-results/all-tests`
to show: `test-results/all-tests`
as it is in the workflow file: https://github.com/Lucra-Sports/ExploratoryTests/actions/runs/2179851359/workflow#L30

This allows to run same command locally, as absolute path is different.

Thank you!

**Comment by ffried** [comment]
I've made a change on `master` that adds another output `unprocessed-command` where all paths are as-is and not resolved. The action also logs both, the resolved invocation as well as this 'unresolved' one.

As for the quotes: The output is now escaped - but using backslashes instead of quotes. Using quotes would require also escaping existing quotes. Using backslashes allows for simply escaping spaces.

I'll release a new version with these changes soon - but it'll require a new major version because I've also updated the node version.
